docs/motivation.md

```diff
@@ -1,3 +1,117 @@
 # Motivation
 
-## **Under Construction**
+## The problem
+
+Consider a Kubernetes cluster that has an application deployment, such as a
+MySQL database, which uses a PersistentVolume (PV) to store its data.  Such
+applications are also referred to as stateful applications or
+[StatefulSets](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/).
+
+A disaster event, be it a natural disaster, such as an earthquake, power failure
+or flooding, etc., or a man-made disaster, such as misconfigurations, malice,
+etc., could leave a Kubernetes cluster unusable.
+
+Until the problems afflicting the cluster are fixed, the affected applications
+may be unstable or plain unusable, resulting in a loss of system availability,
+aka, unscheduled downtime and potential loss of revenue. Businesses that offer
+critical services to its users aim to reduce such unscheduled application
+downtime by preparing a business continuity plan, which includes a disaster
+prevention plan and a disaster recovery (DR) plan.
+
+Note that restarting the application in a different recovery cluster that is not
+prepared for a DR event, may result in an undesirable empty database due to
+dynamic provisioning of a new PV.
```

Comment:
Last part of this statement I would just say something like "Note that restarting the application in a different recovery cluster that is not prepared for a DR event, will not recover the data".

I state this to avoid this early the concept of new PV or empty PV etc. It's just that we loose data and need ways to get it back.